### PR TITLE
Alec/tfac 716 Fully Templatize Email

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

File: graphql/database/cause/causes/cats/social.email.about.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<div>Tab for Cats turns your web browser into a paradise with daily cat background photos. On top of that,  you’ll be helping shelter cats get adopted every time you open a new tab through ad revenue.
```
Comment (kmjennison) [inline graphql/database/cause/causes/cats/social.email.about.md:1]: Remove double space before "you'll": "On top of that,  you’ll"

File: graphql/database/invitedUsers/utils.js [hunk 2]
```diff
@@ -85,6 +86,8 @@ export const verifyAndSendInvite = async ({
       username: encodeURIComponent(invitingUser.username),
       personalMessage: inviterMessage,
       missionId: currentMissionId,
+      ...templateData,
+      title: templateData.title.replace('{{name}}', inviterName),
```
Comment (kmjennison) [inline graphql/database/invitedUsers/utils.js:90]: Can you confirm inviterName is sanitized prior to use here, stripping out any HTML elements? No scripts will load in an email but still don't want an intentionally-manipulated email content.

**Review by kmjennison — CHANGES_REQUESTED** [review]

**Comment by kmjennison** [comment]
@amaliwanag2 If you haven't already, please push to dev for manual testing, too. Thanks!

**Comment by kmjennison** [comment]
@amaliwanag2 Also, please use a descriptive PR title and (as needed) description. Helps a lot in the future when we're looking back at PRs for examples/info.

File: graphql/database/invitedUsers/utils.js (hunk 2 above)
Comment (amaliwanag2) [reply graphql/database/invitedUsers/utils.js:90]: yup we sanitize on the handler of the request

**Pull request merged**